BusinessLayer/Check_EnoughPlayersExist.xaml

```diff
@@ -0,0 +1,109 @@
+<Activity mc:Ignorable="sap sap2010" x:Class="Enough_Players_Exists" this:Enough_Players_Exists.in_ProcessName="msedge" xmlns="http://schemas.microsoft.com/netfx/2009/xaml/activities" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mva="clr-namespace:Microsoft.VisualBasic.Activities;assembly=System.Activities" xmlns:sap="http://schemas.microsoft.com/netfx/2009/xaml/activities/presentation" xmlns:sap2010="http://schemas.microsoft.com/netfx/2010/xaml/activities/presentation" xmlns:scg="clr-namespace:System.Collections.Generic;assembly=mscorlib" xmlns:sco="clr-namespace:System.Collections.ObjectModel;assembly=mscorlib" xmlns:sd="clr-namespace:System.Data;assembly=System.Data" xmlns:this="clr-namespace:" xmlns:ui="http://schemas.uipath.com/workflow/activities" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">

```

Comment:
If there are at least 11 players in extracted data table => just a suggestion, you could make in_dt_PlayerData.RowCount < 11 and go in then sequence (looks better)